azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/service/calling/sdk/CallingSDKWrapper.kt

```diff
@@ -82,6 +82,15 @@ internal class CallingSDKWrapper(
         callingSDKEventHandler.getRemoteParticipantInfoModelFlow()
 
     fun endCall(): CompletableFuture<Void> {
+        val call : Call?
+
+        // The getter for Call throws an exception, so we need to handle this
```

Comment:
here, on back navigation, user get a message "call is not started". As, user want to navigate back, do we want to bubble this error? (User started the call, and want to end on back navigation, I think we do not need to display internal implementation message)

Reply:
It's quiet now.